src/screen/routing.js

```diff
@@ -0,0 +1,23 @@
+import leaflet from "leaflet";
```

Comment:
This entire component could be shifted to `/Components` since it does not return a webpage/JSX.